crates/glue/src/glue.rs

```diff
@@ -2971,20 +2971,7 @@ impl Clone for U4 {
         target_arch = "x86_64"
     ))]
     fn clone(&self) -> Self {
-        let mut answer = unsafe {
-            match self.discriminant() {
-                discriminant_U4::None => core::mem::transmute::<core::mem::MaybeUninit<U4>, U4>(
-                    core::mem::MaybeUninit::uninit(),
-                ),
-                discriminant_U4::Some => Self {
-                    Some: self.Some.clone(),
-                },
-            }
-        };
-
-        answer.set_discriminant(self.discriminant());
-
-        answer
+        *self
```

Comment:
Are we confident all of these changes are correct?

Reply:
I checked a similar change with Richard and he said it was ok.